.github/workflows/ci.yml

```diff
@@ -5,13 +5,22 @@ on:
     branches:
       - "*"
       - "*/*"
+  pull_request:
+    branches:
+      - "*"
+      - "*/*"
 
 jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-13, macos-latest, windows-latest]
-        version: [v2.0.0-alpha.130, v2.0.0-alpha.164, latest, ""]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        version: [v2.0.0-alpha.189, latest, ""]
```

Comment:
You still need to keep the old versions in here.

Reply:
Interesting! Looks like "actually" locally evaluates it differently! (It's my first week trying it out).
I'll add them back!